qcfractal/interface/orm/crank_orm.py

```diff
@@ -1,12 +1,13 @@
 """
-A ORM for Crank
+A ORM for TorsionDrive
```

Comment:
If there is a `torsiondrive_orm.py` file, does that make this one redundant?

Reply:
It does, fixed the history so that it correctly shows a name change and removes the redundant file.